src/locales/en.json

```diff
@@ -70,6 +73,7 @@
   "Select the column index for the following information in the uploaded CSV.": "Select the column index for the following information in the uploaded CSV.",
   "Some of the mapping fields are missing in the CSV: ": "Some of the mapping fields are missing in the CSV: {missingFields}",
   "Something went wrong": "Something went wrong",
+  "Something went wrong, please try again": "Something went wrong, please try again",
   "Something went wrong, Please try again": "Something went wrong, Please try again",
```

Comment:
I think we should use this message instead of adding new with just a character in capitals